src/main/java/core/basesyntax/service/Fruit.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Fruit {
```

Comment:
```suggestion
public interface FruitService {
```

Also, let's keep the consistent naming among all the services.
E.g. let's name `ParserService`, `CsvFileReaderService` and so on. **Don't** name `ProductDaoService`, let it be like it is now.